src/components/SettingsDialog/Components/SettingsInput.scss

```diff
@@ -17,7 +17,7 @@
     font-weight: bold;
```

Comment:
`position: absolute;` can to be removed. ATM the component has a height of 0px in the `ProfileSettings` component. Although the `SettingsInput` in the `BoardSettings` is fine and has the correct height, this is not the case on the `ProfileSettings`.
If there is a reason for using `absolute` positioning, this is fine, but imo it's not needed.